.licenserc.yaml

```diff
@@ -85,6 +85,7 @@ header: # `header` section is configurations for source codes license header.
     - 'ui'
     - '.github/PULL_REQUEST_TEMPLATE'
     - "**/*.prof"
+    - "**/test/integration/etcd/cafile/**"
```

Comment:
Would you please put them into `testdata`, which is ignored by the checker automatically? You can put this folder in any place.